readme.md

```diff
@@ -1,7 +1,7 @@
 # HTML form
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_html-form/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_html-form/report/html_report/)
+- [DEMO LINK](https://Kavon-Skela.github.io/layout_html-form/)
```

Comment:
Don't forget to add demo and test links there:

<img width="1025" alt="Screenshot 2023-02-20 at 11 24 46" src="https://user-images.githubusercontent.com/104769168/220065442-b35272cd-5453-472b-be5a-9937bdcfb28f.png">